lib/foreman_remote_execution_core.rb

```diff
@@ -32,7 +32,9 @@ def self.validate_ssh_log_level!
       raise "Wrong value '#{@settings[:ssh_log_level]}' for ssh_log_level, must be one of #{SSH_LOG_LEVELS.join(', ')}"
     end
 
-    current = if defined?(SmartProxyDynflowCore)
+    current = if defined?(::Proxy::SETTINGS)
+                ::Proxy::SETTINGS.log_level.to_s.downcase
```

Comment:
shouldn't this be fallback for a `SmartProxyDynflowCore` rather than the other way around? :thinking:

Reply:
That's for the case when dynflow core runs inside smart proxy. In that case, settings in `SmartProxyDynflowCore::SETTINGS` are not populated yet

Reply:
Make sense :+1: